temperature_limits.py

```diff
@@ -0,0 +1,11 @@
+coolingType = "PASSIVE_COOLING", "HI_ACTIVE_COOLING","MED_ACTIVE_COOLING"
+
+temperature_limits = {"PASSIVE_COOLING":(0,35),

```

Comment:
strings used in coolingType are used in temperature limits also. If the string are used in more than one place, it is better to define constants and reuse the constants. If not then one change in cooling type and not in temperature limits would lead to failure in future.

Reply:
Thank you for this point.
I have tried to correct this by using constants